src/controllers/occurrence.py

```diff
@@ -57,13 +57,12 @@ def get_all_occurrences(user=None, occurrence_type=None):
         filter = {'user': [user]}
 
     result, code = db.get_all(Occurrence, filter)
-    if result:
-        if code == 200:  # if successful, returns the data
-            # converts rows to dict
-            occurrences = [occurrence.to_dict() for occurrence in result]
-            return occurrences, code
-        return result, code  # else, returns database error and error code
-    return [], 200
```

Comment:
it is necessary to return to an empty list with code 200 when there is no record in the bank and when there was no error